docker-cloud/cloud-swarm/link-azure-swarm.md

```diff
@@ -72,6 +72,30 @@ You need to verify Microsoft Azure terms of use and manually enable your Azure s
 
 You are now ready to deploy a swarm!
 
+## Link an Azure Organization as Global Admin
+
+For members of Azure Organizations, the gobal administrator needs to first link
```

Comment:
Just playing around:

Before you can link your individual Docker Cloud account to your Azure Organization, your global administrator needs to link their account and grant permission to Docker Cloud.

(if you don't change it, s/gobal/global)